backend/src/main/kotlin/be/osoc/team1/backend/entities/Project.kt

```diff
@@ -44,11 +50,11 @@ class Position(
  */
 @Entity
 class Assignment(
-    @OneToOne
+    @ManyToOne
```

Comment:
I'm assuming these changes are just to be able to use `mappedBy`?

Reply:
Yes, because in project it's OneToMany assignments, its impossible to have OneToOne in assignment for a project for example. It needs to be opposites.

Reply:
OK then that's fine, there isn't really another way to do it then, so I'm going to resolve this.